models/biocro/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM pecan/models:develop
+
+RUN mkdir /data
+
+COPY ./model_info.json /work
```

Comment:
Trying to build the image failed, I think because this path needs to be `./models/biocro/model_info.json`?

Reply:
@KristinaRiemer The image should be built in the pecan/models/biocro directory. I made some changes so the image can no longer be built in pecan.